lucille-core/src/main/java/com/kmwllc/lucille/indexer/SolrIndexer.java

```diff
@@ -164,22 +174,35 @@ protected void sendToIndex(List<Document> documents) throws Exception {
         SolrInputDocument solrDoc = toSolrDoc(doc, idOverride, collection);
 
         // if the delete requests contain the ID of this document, send the deletes immediately so
-        // the delete is
-        // processed before this document.
+        // the delete is processed before this document.
 
         if (solrDocRequests.containsIdForDeletion(solrId)
             || solrDocRequests.containsAnyDeleteByField()) {
-          sendDeletionBatch(collection, solrDocRequests);
+          try {
+            sendDeletionBatch(collection, solrDocRequests);
+          } catch (Exception e) {
+            errorsSending.add(e);
+          }
+
           solrDocRequests.resetDeletes();
         }
         solrDocRequests.addDocForAddUpdate(solrDoc);
       }
     }
     for (String collection : solrDocRequestsByCollection.keySet()) {
-      sendAddUpdateBatch(
-          collection, solrDocRequestsByCollection.get(collection).getAddUpdateDocs());
-      sendDeletionBatch(collection, solrDocRequestsByCollection.get(collection));
+      try {
```

Comment:
so, this PR has a bigger consequence than just making sure that failures against one collection don't prevent us from sending to different collections...

it also means that if we have an certain doc ID and there's an update followed by a delete, we might get a failure from the update but still go on to try the delete. Likewise, if there's a delete followed by an update, we might get a failure from the delete, but still go on to try the update. Thinking about this, maybe this new behavior is fine. The important thing is that we never swap an update and a delete, and that's not happening here. But, I wanted to leave this comment just to bring attention to the change -- is this something you considered and also concluded was OK? A code comment recording our conclusion might be helpful for the future.